### DISCUSSION TITLE
"Translate" timer interrupt Arduino uno - Arduino MKR1010Wifi SAMD

### DISCUSSION THREAD

**Discussion by lreinhard7** [khoih-prog/SAMD_TimerInterrupt#16] [Q&A]
Hi, 
Recently, I just started with timer interrupts on Arduino Uno. 
Things work fine but now I want to do something similar on the Arduino MKR1010Wifi Board that uses the SAMD chip. 
Can somebody give me some pointers how to "translate" my code for this new board?
(I use something similar to what is described here in step 11 https://www.instructables.com/Arduino-Bike-Speedometer/)
I found a couple of code example here but miss a bit the explanation for a beginner like me. 
Would appreciate any help I can get. 
Many thanks, 
Ludwig

**Comment by khoih-prog** [comment]
Have a look at the [**RPM_Measure** example](https://github.com/khoih-prog/SAMD_TimerInterrupt/tree/main/examples/RPM_Measure).
It's really hard for any beginner to use interrupt-related code as  it requires certain good knowledge about hardware / firmware.
I'm afraid nobody can have time help you to convert the mentioned code from UNO to SAMD, as the architecture are totally different.
You have to adapt the example code yourself or ask for help in some other forums, such as Arduino, etc.
